### DISCUSSION TITLE
0.16 如何更改區域語言配置

### DISCUSSION THREAD

**Discussion by kkao0** [rime/home#1594] [Q&A] [answered]
在更新日誌中：
> 棄用 weaselt*.dll，增加註冊香港、澳門、新加坡區域語言配置（默認未啓用，需在控制面板/設置中手工添加）；支持簡繁體小狼毫同時使能

我在圖形界面找不到選項。
是不是要在 某個.yaml 修改，如想要更改為繁體香港，要添加的語法是什麼。

**Comment by fxliang** [comment, marked as answer]
不是配置，是在预期的`语言选项`中添加小狼毫输入法，大致效果如下（简中添加繁中台湾为例）
![image](https://github.com/rime/home/assets/4023160/4f68ddee-c7c6-4900-b282-c1583233059d)
![image](https://github.com/rime/home/assets/4023160/ccde8d91-0fc1-49aa-97a1-5bd1bc3f03d1)

**Reply by kkao0** [reply]
感謝
